### ISSUE TITLE
Tagging exercises

### ISSUE THREAD

**Issue by Strauman** [Strauman/exercisebank#2]
Ability to tag exercises and tell sprite only to build exercises with given tags.
Example:
```
%someExercisefile.tex
\tag{exam}
```
Then in main file
```
%main.tex
\sprite[tags={exam}]
```

**Comment by thorstengrothe** [comment]
I played around with the new tagging option of your pre-release and it works pretty well, great work!

I found three things:
1. One **must** tag all problems otherwise the normal `\buildset` command shows nothing.
2. Is it possible to assign multiple tags like so: `tag=easy, sunshine, fun`? I tried it out but it does not work here, only one tag is possible.
3. I personally have a lot of courses and want to reuse my problems. For that I want to use tags because I need to know which problems belong to which courses: `tags=HH17,HH18,HH19`. That way I know what problems I used in the past. I would be very convenient if the user could toggle between showing/not showing tags when the document is compiled, like so:

```
Problem 1
Tags=easy % Toogle on or off
(1a) bla bla
```

**Comment by Strauman** [comment]
Thank you so much for testing!

1. This is really strange! I'm reproducing it nicely, and I'll fix  `\buildset` command ASAP.
1. Multiple tags in lists are possible to implement, and I'll look into it.
1. I'm not really sure what the problem is, but you can build multiple by tags using `\buildtags{taglist}{setname}` as described on page 12 in the docs. If this is not what you meant, would you care to clarify?

**Comment by Strauman** [comment]
@thorstengrothe I'm mentioning you in the changelog for thanks in the docs with a link to your github user. Let me know if this is not okay!

**Comment by thorstengrothe** [comment]
@Strauman 

Ok, let me try to clearify **Point 3**. Say I have 50 part exercises and I used them for different courses over the years. So I tag them for example with `HH17`, `HH194`

Now I make a new exam and would like to see my tags in a first compile. `Exsheets` has such a feature: `debug=true`:

![exsheets_tags](https://user-images.githubusercontent.com/19598706/44631231-77b7e200-a969-11e8-9b06-0637d1610891.png)

This way I now when I used the part problems (HH17, HH194). Now I use your nice `\ShowNumbers` command and can select my part problems. Is that clearer now?

In short: a command `\ShowTags` like your very useful `\ShowNumbers` would be fine! 

Also: Oh it's very nice, that someone mentions me in a changelog, thank you very much!

**Comment by Strauman** [comment]
@thorstengrothe It's clear now! This will be fixed!

**Comment by Strauman** [comment]
@tristelune1 The every exercise-bug should already be fixed in the prerelease-version. See [releases](https://github.com/Strauman/exercisebank/releases)

**Comment by tristelune1** [comment]
Unfortunately it isn't fixed for me. I have downloaded the last version, I have tagged nothing and I compiled with Latex. With version b76 only "Solution" was displayed. With version b77:
   ```
    Exercise 1
        Solution
    Exercise 2
        Solution
```
So only the "headers" are displayed. The description of the problem is not printed. Do you need a small example ? If yes, tell me the best way to send it to you. Perhaps you have a better alternative than Pastebin.

**Comment by Strauman** [comment]
@tristelune1 Thank you so much for testing!

I can't reproduce, so please send me an MWE. The easiest would be writing the code here. Use three backticks, like these: ` ``` YOUR CODE ``` ` to format it nicely! You could also upload code here or, as you mention, use pastebin!

**Comment by Strauman** [comment]
Reproduced with `\makeset{11}{\exclude{secondexercise}{1}}`

**Comment by Strauman** [comment]
@tristelune1 Please open new issue next time, if the title doesn't match :)

**Comment by Strauman** [comment]
@tristelune1 Please try now! If you have issues, please open a new issue if it isn't directly related to tagging exercises :)

**Comment by tristelune1** [comment]
It's fixed, thank you! Great work!

Next time I'll open a new issue, but for me it was related.

**Comment by Strauman** [comment]
@tristelune1 Thanks for your help in these matters, and I'll be sure to mention you in the docs of the next release!
